components/support/utils/src/test/java/mozilla/components/support/utils/DownloadUtilsTest.kt

```diff
@@ -20,32 +22,38 @@ class DownloadUtilsTest {
     fun testGuessFileName_contentDisposition() {
         // Default file name
         assertContentDisposition("downloadfile.bin", "")
-        assertContentDisposition("downloadfile.bin", "attachment")
-        assertContentDisposition("downloadfile.bin", "attachment;")
-        assertContentDisposition("downloadfile.bin", "attachment; filename")
-        assertContentDisposition(".bin", "attachment; filename=")
-        assertContentDisposition(".bin", "attachment; filename=\"\"")
-
-        // Provided filename field
-        assertContentDisposition("filename.jpg", "attachment; filename=\"filename.jpg\"")
-        assertContentDisposition("file\"name.jpg", "attachment; filename=\"file\\\"name.jpg\"")
-        assertContentDisposition("file\\name.jpg", "attachment; filename=\"file\\\\name.jpg\"")
-        assertContentDisposition("file\\\"name.jpg", "attachment; filename=\"file\\\\\\\"name.jpg\"")
-        assertContentDisposition("filename.jpg", "attachment; filename=filename.jpg")
-        assertContentDisposition("filename.jpg", "attachment; filename=filename.jpg; foo")
-        assertContentDisposition("filename.jpg", "attachment; filename=\"filename.jpg\"; foo")
-
-        // UTF-8 encoded filename* field
-        assertContentDisposition("\uD83E\uDD8A + x.jpg",
-                "attachment; filename=\"_.jpg\"; filename*=utf-8'en'%F0%9F%A6%8A%20+%20x.jpg")
-        assertContentDisposition("filename 的副本.jpg",
-                "attachment;filename=\"_.jpg\";" + "filename*=UTF-8''filename%20%E7%9A%84%E5%89%AF%E6%9C%AC.jpg")
-        assertContentDisposition("filename.jpg",
-                "attachment; filename=_.jpg; filename*=utf-8'en'filename.jpg")
-
-        // ISO-8859-1 encoded filename* field
-        assertContentDisposition("file' 'name.jpg",
-                "attachment; filename=\"_.jpg\"; filename*=iso-8859-1'en'file%27%20%27name.jpg")
+
+
```

Comment:
\[ktlint\] WARNING: \[no\-consecutive\-blank\-lines\] Needless blank line\(s\) in column 1